go/runtime/bundle/bundle.go

```diff
@@ -518,7 +518,9 @@ func (bnd *Bundle) Close() error {
 }
 
 // Open opens and validates a runtime bundle instance.
-func Open(fn string) (*Bundle, error) {
+func Open(fn string, opts ...OpenOption) (*Bundle, error) {
+	options := NewOpenOptions(opts...)
+
 	r, err := zip.OpenReader(fn)
```

Comment:
Unrelated to this PR, but just noticed that if this fails below, the reader is never closed.